changelogs/unreleased/gh-6247-broken-error-message.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Fix a error message in mvcc (gh-6247)
```

Comment:
*an

* Fix an incorrect error message that may appear during DML requests in MVCC mode (gh-6247).

IMHO changelogs must be user-friendly:)